deploy/README.md

```diff
@@ -161,16 +162,57 @@ kubectl -n sumologic create secret generic sumologic \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 ```
 
-Apply `fluentd-sumologic.yaml` manifest with following command:
+Get the `fluentd-sumologic.yaml` manifest with following command:
 
 ```sh
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" | \
+sed 's/\$NAMESPACE'"/sumologic/g"
```

Comment:
```
sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
```
to save the output to the file `fluentd-sumologic.yaml`

Reply:
this is only needed if the user wants to customize. if not, they should still to use the previous command that applies `kubectl apply` altogether which is one less step than running two commands separately